docs/bmad/stories/48.10.layout-modi.md

```
@@ -0,0 +1,420 @@
+### Integration Verification
+
+- **IV1:** LagekarteView Component akzeptiert `mode` Prop und passt Darstellung an
+- **IV2:** Bestehende Lagekarte-Funktionalität bleibt in Standard-Modus unverändert
+- **IV3:** TanStack Router validateSearch funktioniert korrekt für mode-Parameter
+
+---
+
+## Tasks / Subtasks
+
+- [ ] **Task 1: TypeScript Types für Layout-Modi** (AC: 1-3, IV1)
+    - [ ] Definiere `LagekarteMode` Type: `'standard' | 'fullscreen' | 'presentation'`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)